ftplugin/markdown.vim

```diff
@@ -0,0 +1,32 @@
+
+
+" plasticboy/vim-markdown {{{1
+
+hi! link htmlH1 DraculaPurpleBold
+
+hi! link htmlItalic DraculaYellowItalic
+hi! link mkdItalic DraculaYellowItalic
+
+hi! link htmlBold DraculaOrangeBold
+hi! link mkdBold DraculaOrangeBold
+
+hi! link htmlBoldItalic DraculaOrangeBoldItalic
+hi! link mkdBoldItalic DraculaOrangeBoldItalic
+
+hi! link mkdRule DraculaComment
+
+hi! link mkdBlockquote DraculaYellowItalic
+
+hi! link mkdListItem DraculaCyan
+
+hi! link mkdCode DraculaGreen
```

Comment:
In standard dracula, this is `DraculaGreenItalic`

Reply:
Hm... Not sure why I made the base syntax italic. The spec says it should be non-italic. 

I'll fix the base implementation. Really good catch!